.github/workflows/codecov.yml

```diff
@@ -39,12 +39,12 @@ jobs:
       #----------------------------------------------
       - name: Install dependencies
         if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
-        run: poetry install --no-interaction --no-root
+        run: poetry install --no-interaction --no-root --all-extras --with dev
       #----------------------------------------------
-      # install your root project, if required
+      # install pycytominer
       #----------------------------------------------
       - name: Install library
-        run: poetry install --no-interaction
+        run: poetry install --no-interaction --only-root --all-extras
```

Comment:
I agree this was confusing. It technically sped up the workflows a small amount, but I don't think it's worth the loss in clarity. This approach was already going to be removed in #342, but I stripped it during this PR too in case that other PR is changed/delayed.